### PR TITLE
local cluster proposal

### PR DESCRIPTION
**Pull request by wwitzel3**
Signed-off-by: Wayne Witzel III <wayne@riotousliving.com>

### CODE REVIEW

File: proposals/20191022-local-cluster-mgmt.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+### Goals
+ - KIND cluster verbs: create, delete, stop, start
+ - Listing KIND clusters.
+ - Single click to switch Octant  context.
+
+### Non-Goals
+ - Managing non-KIND clusters.
+
+### Optional
+  - Install KIND from Octant.
```
Comment (wwitzel3) [inline proposals/20191022-local-cluster-mgmt.md]: bundle KIND with octant

**Comment by wwitzel3** [comment]
current octant will not start if there is no cluster / kubeconfig

**Comment by bryanl** [comment]
Before this can be worked on, Octant will required the ability to start with a kubeconfig file.

**Comment by wwitzel3** [comment]
Revisit after #362

**Pull request closed without merging**